Inseok_45/middlewares/validate-jwt.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+
+
+const validateToken = async (req, res, next) => {
+    const token = req.headers.authorization;
+
+    if(!token) return res.status(403).json({ message: "Can't Log In" });
+
+    const verified = jwt.verify(token, process.env.JWT_SECRETKEY);
+    req.user = verified.id;
```

Comment:
```suggestion
    const verified = await jwt.verify(token, process.env.JWT_SECRETKEY);
    
    const user = await userService.getUserById(decoded.id)

    if (!user) {
	const error = new Error('USER_DOES_NOT_EXIST')
	error.statusCode = 404
		
	return res.status(error.statusCode).json({message: error.message})
    }
    
    req.user = user.id;
```
토큰 검증 후에는 해당 토큰의 주인이 여전히 서버에서 유효한 회원인지를 검증하는 과정 또한 필요합니다. 인가처리는 보안상 민감한 부분이므로 로직이 꼼꼼히 작성될수록 좋습니다-!